build.gradle

```diff
@@ -130,6 +144,8 @@ task unitTest(type: Test) {
     }
 }
 
+// TODO(gjordan): turn off doclint to remove "no @return value" warnings
+// https://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html
```

Comment:
Or fix the javadoc :)

Reply:
A valid option – but unnecessary @return tags *feel* like noise to me, and I think Google's Java style guide agrees, since both their examples omit the @return block ([ref](https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line)).

The fact that it's just a warning, and not an error, suggests Javadoc feels at least somewhat similarly :)